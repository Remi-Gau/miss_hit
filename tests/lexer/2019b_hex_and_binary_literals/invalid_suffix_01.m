0x12s12
