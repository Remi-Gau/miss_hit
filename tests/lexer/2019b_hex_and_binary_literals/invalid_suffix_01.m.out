In invalid_suffix_01.m, line 1
| 0x12s12
|       ^ [31;1mlex error[0m: suffix must be 8, 16, 32, or 64
invalid_suffix_01.m: lexed with errors
MISS_HIT Debug Summary: 1 file(s) analysed, 1 error(s)
